extension/index.googleFunction.js

```diff
@@ -8,6 +8,17 @@ const logger = utils.getLogger()
 exports.extensionTrigger = async (request, response) => {
   try {
     const authToken = auth.getAuthorizationRequestHeader(request)
+    if (!request?.body?.resource) {
```

Comment:
What returns when `auth.getAuthorizationRequestHeader` throws an error ? Any test case ?

Reply:
it should be caught here
https://github.com/commercetools/commercetools-adyen-integration/blob/ffc9c0db114fc33e3d730832fff0539acc04ca9e/extension/index.googleFunction.js#L31

so far we don't have unit test for google cloud HTTP function. It may need another unit test framework
https://www.deloittedigital.at/google-cloud-functions-testing-part-1/

Reply:
Added a unit test for GCP extension trigger
https://github.com/commercetools/commercetools-adyen-integration/pull/630/commits/3540c47391dd9c16aa8f74df602a268d7f4df92d